test/helper_non-windows_test.go

```diff
@@ -0,0 +1,10 @@
+// +build !windows
```

Comment:
> bsd/darwin is not unix

@petemoore, afaik the BSDs including OS X actually derive from the original unix :)
OS X has certainly been unix certified at times... Regardless, they are all consider unix-like, just like linux (which also isn't unix).
We could also call it `_posix`, but I don't enjoy dashes in golang filenames, it just feels wrong...